lfucg-exactions/client/js/components/AccountForm.js

```diff
@@ -183,14 +186,19 @@ class AccountForm extends React.Component {
                                         </div>
                                     </div>
                                 </fieldset>
-                                <button disabled={!submitEnabled} className="btn btn-lex">Submit</button>
-                                {!submitEnabled ? (
-                                    <div>
-                                        <div className="clearfix" />
-                                        <span> * All required fields must be filled.</span>
-                                    </div>
-                                ) : null
-                                }
+                                <div className="col-xs-8">
```

Comment:
Could this section be extracted into a common component instead?

Reply:
Some could, but the onSubmit functions referenced differ on some pages (particularly PlatForm).